pixart.js

```diff
@@ -1 +1,52 @@
 
+var controls = document.querySelector(".controls")
+var form = document.querySelector("#form")
+var field = document.querySelector("#color-field")
+var button = document.querySelector("#set-color")
+var brush = document.querySelector(".brush")
+
+var numOfSwatches = 3
+for (var i = 0; i <numOfSwatches; i++){
+  var swatch = document.createElement("div")
+  swatch.setAttribute("class", "swatch")
+  swatch.style.float = "left"
+  swatch.style.margin = "0 2px"
+  controls.insertBefore(swatch, form);
+}
+
+form.style.clear = "both"
+
+//When I click the "Set Color" button, it should change the color of the "brush" box to the color I specify in the input field. 
+var swatches = document.querySelectorAll(".swatch")
+var swatchCounter = 0
+button.addEventListener("click", function (event){
+  if(swatchCounter >= numOfSwatches){
+    swatchCounter = 0
+  }
+  event.preventDefault();
+  brush.style.background = field.value
+  swatches[swatchCounter].style.background = field.value
+  swatchCounter++
+})
```

Comment:
cool solution to the swatch, i saw ryans solution for it and like that one as well, i would check that one out when you have a chance